### ISSUE TITLE
All process update the network and then sync the grad?

### ISSUE THREAD

**Issue by nizhihao** [TianhongDai/hindsight-experience-replay#14]
Hi, I have the doubt. 
In this distributed RL, because of OS scheduling, every process will have the near state and do the same thing?
so all process will update the network and then sync the grad together?  like sync algorithm A2C？
thanks very much.

**Comment by TianhongDai** [comment]
Yes, it can be refereed as using a very large batch size for the training.
